app/controllers/api/v1x0/requests_controller.rb

```diff
@@ -14,11 +14,29 @@ def show
       end
 
       def index
+        if params[:approver]
+          return index_by_approver
```

Comment:
why `return` here? shouldn't we support other filtering (decision, state etc) for an approver?
Please consider the filtering together with #104

Reply:
New changes are ready, will wait for #104 is merged.